### ISSUE TITLE
Quests are listed as Quest - Quest - <Expansion>

### ISSUE THREAD

**Issue by prathmell** [Cidan/BetterBags#183]
### Addon Version

v0.1.6

### World of Warcraft Version

Retail

### Describe the bug

In the bag, quests are shown as Quest - Quest - <Expansion> the phrase "Quest - "i s duplicated.  I also don't think you need the " - " either but that's an aesthetic decision. IMO there's too much white space

### Steps to reproduce

Open bag.  Mouse over a quest category.

**Comment by Cidan** [comment]
I agree, Quest - Quest is unique because both the type and subtype are Quest. This should be deduped.
